src/app/Fake.IO.FileSystem/Directory.fs

```diff
@@ -0,0 +1,49 @@
+namespace Fake.IO.FileSystem
+
+open System.IO
+
+module Directory =
+
+    /// Creates a directory if it does not exist.
+    let CreateDir path = 
+        let dir = DirectoryInfo.OfPath path
+        if not dir.Exists then 
+            // TODO: logfn "Creating %s" dir.FullName
+            dir.Create()
+        else () //TODO: logfn "%s already exists." dir.FullName
+
+    /// Checks if the given directory exists. If not then this functions creates the directory.
+    let inline ensure dir =
+        if not (Directory.Exists dir) then
+            Directory.CreateDirectory dir |> ignore
+            
+    let isDirectory path = Path.isDirectory path
```

Comment:
`Directory.isDirectory path` reads a bit strange. Though I have no suggestion how to improve. Maybe just leave `Directory.Exist` and drop this?

Reply:
Obsolete to use Path.IsDirectory?

Reply:
Obsolete to use Path.IsDirectory?